### ISSUE TITLE
grruby.so location is ambiguous. 

### ISSUE THREAD

**Issue by kojix2** [SciRuby/rubyplot#55]
Hi
grruby.so location is ambiguous. 

By default, `gems/rubyplot -0.1 .pre.a1/lib/grruby/grruby.so` is generated.

But this file is not read.
rubyplot reads `gems/rubyplot -0.1 .pre.a1/lib/grruby.so`

**Comment by kojix2** [comment]
So, You have to move `grruby.so` or create a symbolic link, before using Rubyplot.

@alishdipani
Could you take a look at this issue so that more people can enjoy Rubyplot?

**Comment by alishdipani** [comment]
@kojix2  Yes, after gem istallation GR backend is not working due to `grruby.so`.
Thanks for pointing this out, I will look into it soon.

**Comment by kojix2** [comment]
So you're thinking about the location of grruby.so because you might add GR3 and GKS modules, right?

**Comment by v0dro** [comment]
Yes that's right. I think adding a fallback using `LoadError` should do the trick for now. Sorry I could not focus on rubyplot for a while until now. Will get back to it soon.
